express/scripts/instrument.js

```diff
@@ -745,6 +752,67 @@ loadScript(martechURL, () => {
     }
   }
 
+  // Frictionless Quick Actions tracking events
+
+  function sendEventToAdobeAnaltics(eventName) {
```

Comment:
Couldn't we just re-use the code L706+ that already tracks via alloy? We could extract a common method for it essentially